lib/screen/interactive_postcard/postcard_detail_state.dart

```diff
@@ -129,11 +142,22 @@ extension PostcardDetailStateExtension on PostcardDetailState {
   }
 
   bool get canDoAction {
+    assetToken?.postcardMetadata.counter;
```

Comment:
can you explain what this line uses for?

Reply:
my mistake, deleted it